packages/sdk-socket-server-next/src/protocol/handleCheckRoom.ts

```diff
@@ -33,8 +33,11 @@ export const handleCheckRoom = async ({
 
   const room = io.sockets.adapter.rooms.get(channelId);
   const occupancy = room ? room.size : 0;
-  const channelOccupancy =
-    (await pubClient.hget('channels', channelId)) ?? undefined;
+
+  const sRedisChannelOccupancy = await pubClient.hget('channels', channelId);
```

Comment:
So this can be either a JSON string or a number (old format)? This function assumes this will always be a number. Does this code path need to handle both cases like the other places?